app/components/ActionLinks/index.js

```diff
@@ -89,6 +92,16 @@ const ActionLinks = ({
             Global search
           </ActionLink>
         )}
+
+        {isAddBulkAppointments && addBulkAppointmentsUrl && (
```

Comment:
where is URL defined, we don't need a config param for this as we know it hangs of PSH I assume?

Reply:
OK. I'll pass the PSH url to the front end and adjust the link values accordingly.